vochain/indexer/vote.go

```diff
@@ -214,50 +210,42 @@ func (s *Indexer) GetEnvelopeHeight(processID []byte) (uint64, error) {
 	return uint64(height), err
 }
 
-// ComputeResult process a finished voting, compute the results and saves it in the Storage.
+// finalizeResults process a finished voting, get the results from the state and saves it in the indexer Storage.
 // Once this function is called, any future live vote event for the processId will be discarted.
-func (s *Indexer) ComputeResult(processID []byte) error {
+func (s *Indexer) finalizeResults(processID []byte) error {
 	height := s.App.Height()
-	log.Debugf("computing results on height %d for %x", height, processID)
+	log.Debugw("finalize results", "processID", hex.EncodeToString(processID), "height", height)
```

Comment:
I think the concept of finalize results should be better explained.

Reply:
What you mean? This is a debug message. Its scope is to provide technical information to the developer.

Reply:
Sorry I was not referring to this debug message in particular, but added the message here. I was referring to the concept of finalizing the results. It is a new concept at least for me and was not easy to understand the change at first glance. That is why I suggested to add more context, maybe on top of the function.